azure-pipeline.yml

```diff
@@ -83,7 +83,13 @@ stages:
           displayName: 'Use .NET Core sdk'
           inputs:
             packageType: sdk
-            version: 5.0.400
+            version: 5.0.x
+            installationPath: $(Build.SourcesDirectory)/.dotnet
+        - task: UseDotNet@2
+          displayName: 'Use .NET Core sdk'
+          inputs:
+            packageType: sdk
+            version: 6.0.100
```

Comment:
maybe we can do a `6.0.x` here so we are not pinning on a very specific version.

Reply:
nvm, this might need to be 6.0.100 for now.

Reply:
Yup, tried `.x` and we hit our Integration test problem. Probably going to have to stick here until we figure out what's going on there.